core/loadpoint.go

```diff
@@ -1450,6 +1459,45 @@ func (lp *LoadPoint) Update(sitePower float64, cheap bool, batteryBuffered bool)
 		err = lp.setLimit(targetCurrent, required)
 	}
 
+	// reset WakeUpCalled mode off
+	if mode == api.ModeOff && lp.wakeUpTimer.active {
+		lp.log.DEBUG.Printf("reset WakeUpTimer - mode off - Active:%ds", lp.wakeUpTimer.duration())
+		lp.wakeUpTimer.Reset()
+	}
+	// reset WakeUpCalled then car is charging again
+	if lp.enabled && lp.status == api.StatusC && (lp.wakeUpTimer.called || lp.wakeUpTimer.active) {
```

Comment:
das reicht der check auf Status C

Reply:
hatte den Status C mit enabled False bei mir aber so wo ich drüber nachdenke könnte sein das es an der 2. EVCC Instanz lag. ?!?

Reply:
Kann eigentlich nur sowas sein, denn das ist nahezu ausgeschlossen da dies direkt eine "out of sync" Warning auswerfen würde. Status C und `!enabled` würde sich logisch widersprechen.

Reply:
Und B und !enabled würde aber gehen oder?

Reply:
wie oben bitte nochmal drüber schauen - aber denke das es jetzt so passt.